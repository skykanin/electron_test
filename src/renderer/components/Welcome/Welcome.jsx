import React from 'react'

export default class Welcome extends React.Component {
    render() {
        return(
            <h1>
                Hello from {this.props.compiler} and {this.props.framework}!
            </h1>
        )
    }
}