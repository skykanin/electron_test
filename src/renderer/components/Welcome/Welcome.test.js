import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Welcome from './Welcome'

test('Welcome component should render as expected', () => {
    const wrapper = shallow(<Welcome compiler="Javascript" framework="React" />);
    const tree = toJson(wrapper);
    expect(tree).toMatchSnapshot();
})