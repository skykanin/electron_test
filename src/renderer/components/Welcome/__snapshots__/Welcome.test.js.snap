// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome component should render as expected 1`] = `
<h1
  className="welcome"
>
  Hello from 
  Javascript
   and 
  React
  !
</h1>
`;
